op-chain-ops/deployer/integration_test/apply_test.go

```diff
@@ -56,7 +59,7 @@ func (d *deployerKey) String() string {
 func TestEndToEndApply(t *testing.T) {
 	kurtosisutil.Test(t)
 
-	lgr := testlog.Logger(t, slog.LevelInfo)
+	lgr := testlog.Logger(t, slog.LevelDebug)
```

Comment:
Do we want to keep this change?

Reply:
Let's keep it for now so we can get this merged. We can easily revert later if it's a problem.